L1Trigger/L1THGCalUtilities/python/caloTruthCells_cff.py

```diff
@@ -0,0 +1,89 @@
+import FWCore.ParameterSet.Config as cms
+
+from L1Trigger.L1THGCalUtilities.caloTruthCellsProducer_cfi import *
+from L1Trigger.L1THGCalUtilities.hgcalTriggerNtuples_cfi import *
+
+caloTruthCells = cms.Sequence(caloTruthCellsProducer)
+
+# Add the "full truth" multiclusters (each cluster made with all cells belonging to one gen particle) to the ntuples
+
+ntuple_multiclusters_fulltruth = ntuple_multiclusters.clone(
```

Comment:
I would put all the ntuple configuration in a different config file.
Also, it will clash with the `hgcalTriggerChains` configuration. At some point all this should be integrated as part of the `hgcalTriggerChains`

Reply:
For this, I'd like to either discuss with you or just ask you to reorganize the configurations to suit the overall design.

Reply:
The main issue here is that it couples the producer and the ntuplizer. I think the first step would be to keep them separated in different config files. Then we could think at a better design.

Reply:
Ntuples part split into caloTruthCellsNtuples_cff.py.